roles/ceph-update/tasks/ceph_owner.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
tests on tesla show that creating cinder volume failed until restart cinder services.
I will move the task of restarting cinder services to cinder-common role.